x-pack/plugins/security_solution/public/timelines/components/timeline/tabs/shared/use_timeline_control_columns.tsx

```diff
@@ -78,26 +84,38 @@ export const useTimelineControlColumn = ({
             />
           );
         },
-        rowCellRender: (props: EuiDataGridCellValueElementProps & TimelineDataGridCellContext) => {
+        rowCellRender: (
```

Comment:
```
const eventItem = events ? events[props.rowIndex] : null;
```

Reply:
Thanks @michaelolo24 . This was fixed here ( 67d3e64dc2236690f2d8067141f384a87a781e58 ) already. See if that method works.

Also, returning null causes the error with `EuiDataGrid` so i am returning an empty element.